assets/js/services/GuardedRoute.tsx

```diff
@@ -0,0 +1,9 @@
+import React from "react";
+import { Navigate, Outlet } from "react-router-dom";
+
+const GuardedRoute = () => {
+  const auth = localStorage.getItem("jwt") != null;
```

Comment:
- I would change the variable name to `isAuthenticated`
- I would move it to function like `isUserAuthenticated` or something like this. It could be in file `services/index.js.

Reply:
Done ✅